src/env/node/git/localGitProvider.ts

```diff
@@ -3320,21 +3322,28 @@ export class LocalGitProvider implements GitProvider, Disposable {
 				let onRef;
 				let summary;
 				let message;
+				let wipSummary;
+				let matchSummary;
+
 				const match = stashSummaryRegex.exec(s.summary);
 				if (match?.groups != null) {
+					matchSummary = match.groups.summary.trim();
 					onRef = match.groups.onref;
+					wipSummary = `WIP on ${onRef}`;
 					if (match.groups.wip) {
-						message = `WIP: ${match.groups.summary.trim()}`;
-						summary = `WIP on ${onRef}`;
+						message = `WIP: ${matchSummary}`;
+						summary = wipSummary;
 					} else {
-						message = match.groups.summary.trim();
-						summary = message.split('\n', 1)[0] ?? '';
+						message = matchSummary;
+						summary = message ? message.split('\n', 1)[0] : wipSummary;
```

Comment:
if the message is empty should we also set it to the wipSummary? In this case, _message is `undefined` and _summary is `WIP on development`
![image](https://user-images.githubusercontent.com/42983615/180840487-ea1f092c-818f-4df0-a48b-5e35a6638f0b.png)

Reply:
Nice catch! I had logic set up for this but it looks like it may have gotten lost when I rebased today. Mind trying it again?